### ISSUE TITLE
New Kotlin sourceset layout breaks Moko resources

### ISSUE THREAD

**Issue by DigitalKoi** [msa1422/KMM-PetsApp#2]
The new sourceset layout brought on by the Kotlin update to 1.8 seems to break Moko resources

`Execution failed for task ':petsearch-shared:resources:packageDebugResources'.`

**Comment by msa1422** [comment]
This issue [#410](https://github.com/icerockdev/moko-resources/issues/410) has already been reported in the Moko-resources Github repo and it has been added to the next milestone of the library. In my case, this issue arises with a combination of K1.8.0, AGP-7.4.1 & kotlin.mpp.androidSourceSetLayoutVersion=2. 
To avoid the issue, I have kept the AGP version to 7.4.0 as of now and I am desperately hoping that the primary maintainer of the Moko-resources (@Alex009) updates the library soon.
